src/commands/farm.rs

```diff
@@ -347,3 +412,23 @@ fn syncing_progress_bar(current_block: u64, total_blocks: u64) -> ProgressBar {
     pb.set_position(current_block);
     pb
 }
+
+fn not_yet_processed_block_nums_stream(
```

Comment:
these are actually not new blocks. Syncing maybe ahead of this. The name is long, I agree, but it's clear. 
`new_blocks` are confusing imo. 2 months later we can forget the distinction and it may not make sense

Reply:
Okay, so be it!